Modules/_abc.c

```diff
@@ -67,6 +67,8 @@ typedef struct {
     uint64_t _abc_negative_cache_version;
 } _abc_data;
 
+#define _abc_data_CAST(op)   ((_abc_data *)(op))
```

Comment:
I'm curious about the decision to use 3 spaces between the cast and macro name in these PRs. Is there any reasoning behind it, or is it just arbitrary?

Reply:
It's to align to the next tab

Reply:
which... in this case is wrong :D

Reply:
(I'll check the indentations because I've done a lot of C/C xD)

Reply:
Cool. WDYM by align to the next tab? The indentation in the next function body? (Sorry if that's a silly question, it's late for me.)

Reply:
Oh no, it's not a silly question. You see, when I define multiple macros, or when I use imports, I do

```C
#include <...>			// XXX
#include <.........> 	// YYY
```

where `//` are aligned vertically. To ease alignment, I don't use spaces but only tabs after `>`. That way, it's always on a multiple of 4 that `/` is placed. Same for macros:

```C
#define NAME			WHAT1
#define NAME........	WHAT2
```

Generally those macros do the same work and thus it's always in prevision of future similar macros. It's also a bit easier to read the macro body when it's not stick to the macro declaration, e.g.:

```C
#define DO_SOMETHING_(op) ((PTR *)(op))
```

vs

```C
#define DO_SOMETHING_(op)	((PTR *)(op))
```

Reply:
Ah, got it. Thanks for clarifying! This is why we invented formatters, though :smile:

> It's also a bit easier to read the macro body when it's not stick to the macro declaration

I actually see the opposite! For me, the second one here takes an extra second to process. I don't think it matters much for your fleet of PRs, though--I was just curious. (But you should probably aim for <=4 when there's just one macro.)

Reply:
> I don't think it matters much for your fleet of PRs

And this is only the first wave!

Reply:
I've tagged you on the issue.

Reply:
Will take a look tomorrow (unless I'm asleep all day, otherwise I'll look at it on Saturday).